client/app/cloud/project/compute/regions/regions.less

```diff
@@ -0,0 +1,5 @@
+.cloud-compute__regions {
+  .regions-info-message .oui-message {
+    margin: 0 0 1.6rem;
```

Comment:
Could a class (https://getbootstrap.com/docs/4.1/utilities/spacing/ ) be applied to the given elements instead ?

Reply:
@marie-j , not possible. I have to apply a custom margin to div inside oui-message not to oui-message itself. The component does not support a custom class for it.